dapp-manager/src/main/kotlin/jp/co/soramitsu/dapp/manager/controller/RegistrationInitialization.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package jp.co.soramitsu.dapp.manager.controller
+
+import com.d3.commons.registration.RegistrationServiceEndpoint
+import com.d3.commons.util.createPrettySingleThreadPool
+import io.reactivex.schedulers.Schedulers
+import jp.co.soramitsu.dapp.manager.config.DAPP_MANAGER_NAME
+import jp.co.soramitsu.dapp.manager.registration.DappRegistrationStrategy
+import mu.KLogging
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class RegistrationInitialization(
+    @Autowired
+    private val registrationPort: Int,
+    @Autowired
+    private val dappRegistrationStrategy: DappRegistrationStrategy
+) {
```

Comment:
You don't need @Autowired for constructor parameters injection

Reply:
Thanks, seems so, done